### DISCUSSION TITLE
Token based 2FA to use with Authy/Google Authenticator

### DISCUSSION THREAD

**Discussion by mkschulze** [GenieFramework/Genie.jl#583] [Ideas]
Hello,

it would be great to have an additional factor for authentication I think. I often see it with webapps, seems to make sense.

Seems like OpenIDconnect.jl can handle token based auth, is this what I mean? 
https://github.com/tanmaykm/OpenIDConnect.jl#token-requests

There is an issue already here: https://github.com/GenieFramework/GenieAuthentication.jl/issues/8
and also a mention about token from @hhaensel here: https://github.com/GenieFramework/GenieAuthentication.jl/issues/7

**Comment by hhaensel** [comment]
I'll have a look into it.
Currently I'm building  token-baded SSO authentication for one of my Stipple Apps.
